packages/cdk-blue-green-container-deployment/src/__tests__/ecs-service.test.ts

```diff
@@ -91,4 +91,30 @@ describe('EcsService', () => {
       );
     });
   });
+
+    describe('with execute command', () => {
```

Comment:
In the default test, what does `EnableExecuteCommand` resolve to? Can you add that to the "with default props" test?